flows/pom.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
the main pom should also list `flows` as a module?

Reply:
Yes, it should. Added.